backend/app/api/routes/projects.py

```
@@ -2,153 +2,289 @@
+        logger.info(
+            "Project retrieved successfully",
+            project_id=str(project_id),
+            tenant_id=str(tenant_id),
+            user_id=str(current_user.id),
+        )
+
+        return response
+
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)